x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -405,9 +406,10 @@ const serviceThroughputRoute = createApmServerRoute({
       comparisonStart,
       comparisonEnd,
     } = params.query;
-    const searchAggregatedTransactions = await getSearchAggregatedTransactions(
-      setup
-    );
+    const searchAggregatedTransactions = await getSearchAggregatedTransactions({
+      ...setup,
+      kuery,
+    });
```

Comment:
I created https://github.com/elastic/kibana/issues/103878 to remove `start` and `end` from setup. To prepare for that it might be better to pass the entire `params`:

```suggestion
    const searchAggregatedTransactions = await getSearchAggregatedTransactions({
      params,
      apmEventClient,
    });
```